libs/application/templates/financial-aid/src/lib/types.ts

```diff
@@ -5,6 +5,11 @@ export enum DataProviderTypes {
   NationalRegistry = 'NationalRegistryProvider',
 }
 
+export enum TwoTypeAnswers {
```

Comment:
Can we renam this? Something like ApproveOptions or along that line would be better